compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TaintAnalyzer.java

```diff
@@ -1819,8 +1925,63 @@ private void analyzeInvocation(BLangInvocation invocationExpr) {
         taintedStatus = returnTaintedStatus;
     }
 
+    private void updateArgTaintedStatus (BLangInvocation invocationExpr, List<TaintedStatus> argTaintedStatusList) {
+        if (!argTaintedStatusList.isEmpty()) {
```

Comment:
Best practice is to fail fast, inverting this if. Then code becomes more readable.